src/syft/core/plan/translation/torchscript/plan_translate.py

```diff
@@ -0,0 +1,108 @@
+# stdlib
+from collections import OrderedDict
+from typing import Any
+from typing import List
+
+# third party
+import torch as th
+
+# syft absolute
+from syft.core.plan import Plan
+from syft.core.plan.plan_builder import PLAN_BUILDER_VM
+from syft.core.plan.plan_builder import ROOT_CLIENT
+from syft.lib.python.collections import OrderedDict as SyOrderedDict
+from syft.lib.python.dict import Dict
+from syft.lib.python.list import List as SyList
+from syft.logger import traceback_and_raise
+
+# syft relative
+from ....pointer.pointer import Pointer
+from .plan import PlanTorchscript
+
+
+def translate(plan: Plan) -> PlanTorchscript:
+    """Translates Syft Plan to torchscript"""
+
+    class PlanWrapper(th.nn.Module):
+        """
+        Plan needs to be executed with kwargs while torchscript needs to be executed with args,
+        so we need to have kwarg names inside torchscript to pass them into Plan.
+        We use nn.Module to store input kwarg names inside torchscript.
+        """
+
+        def __init__(self, kwarg_names: List[str]):
+            _kwarg_names: List[str]
+            super(PlanWrapper, self).__init__()
+            self._kwarg_names = kwarg_names
+
+        def forward(self, *args: Any) -> Any:
```

Comment:
maybe execute is a better name, as people may associate `.forward` with `torch.nn.module.forward`

Reply:
it really is nn.Module :) 
This wrapper is a workaround to store extra state (kwarg names for Plan) that can't be passed as argument into torchscript (note: it's possible to pass dict to torchscript, but dict values need to have same type, i.e. you'd need to flatten model params list into this dict)

Reply:
Oh wow, it took me a few minutes but now I get it. I would document this extremely clearly, not just in the init but also in the forward method, because it makes sense once you get it but I think it could potentially confuse many people.